backend/.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+# This workflow runs ESLint, Prettier, unit tests, and e2e tests.
+name: CI
+
+on:
+  push:
+    branches: [main]
```

Comment:
The ci should run on all branches except main and develop. You will need to update the condition.